tests/HashMapProperties.hs

```diff
@@ -49,6 +49,14 @@ pFoldable :: [(Int, Int)] -> Bool
 pFoldable = (L.sort . Foldable.foldr (:) []) `eq`
             (L.sort . Foldable.foldr (:) [])
 
+pHashable :: [(Key, Int)] -> [Int] -> Int -> Property
+pHashable xs is salt =
+    x == y ==> hashWithSalt salt x === hashWithSalt salt y
```

Comment:
Well, *different*. `fromList [(1,2),(1,3)]` and `fromList [(1,3),(1,2)]` are the same if you ask from `(==)`. And there is a `Hashable` law:

> If two values are equal according to the `==` method, then applying the `hashWithSalt` method on each of the two values must produce the same integer result if the same salt is used in each case.